packages/block-editor/src/store/test/selectors.js

```diff
@@ -3043,7 +3043,7 @@ describe( 'selectors', () => {
 					byClientId: new Map(
 						Object.entries( {
 							block1: { name: 'core/test-block-ancestor' },
-							block2: { name: 'core/block' },
+							block2: { name: 'core/group' },
```

Comment:
This test was failing because it tries to insert a block within a child of a `core/block` block, this wouldn't be possible by a user in trunk and it fails now that the PR enforces the block editing mode at selector level.

I've switched it to a group instead.